internal/graph/check.go

```diff
@@ -123,7 +123,7 @@ type checkOutcome struct {
 }
 
 type LocalChecker struct {
-	ds                 storage.RelationshipTupleReader
+	ds                 storage.Reader
```

Comment:
Same comment as in https://github.com/openfga/openfga/pull/1258/files#r1441147756